assets/src/dashboard/app/views/myStories/karma/myStories.karma.js

```diff
@@ -56,6 +56,74 @@ describe('Grid view', () => {
     return stories;
   }
 
+  async function focusOnGridByKeyboard() {
+    let limit = 0;
+    const gridContainer = fixture.screen.getByTestId('dashboard-grid-list');
+
+    while (!gridContainer.contains(document.activeElement) && limit < 20) {
```

Comment:
What's the significance of `20`? Could this number be determined from the length of stories to make it less "magic".

Reply:
No, the limit is just intended as a safety against infinite loops. This is lifted from the explore templates karma test. I just thought it was cleaner than adding 'tab' keyboard presses an arbitrary number of times and not including a safety limit.

Reply:
oh, right on!